src/views/v2/components/messages.php

```diff
@@ -29,6 +29,7 @@
 <div <?php tribe_classes( $classes ); ?>>
 	<?php foreach ( $messages as $message_type => $message_group ) : ?>
 		<div class="tribe-events-c-messages__message tribe-events-c-messages__message--<?php echo esc_attr( $message_type ); ?>" role="alert">
+			<?php $this->template( 'components/messages/' . esc_attr( $message_type ) . '/icon' ); ?>
```

Comment:
let's call this `components/messages/notice-icon` instead, since we don't have a `notice` template (the folder structure follows the templates that are included).